workshops/collaborative_sketch/README.md

```diff
@@ -122,9 +120,21 @@ var config = {
 }
 ```
 
-This section of code creates a new `object` in JavaScript named `config`.  You can think of `objects` like containers for things like text or numbers which we call `properties`. 
+This section of code creates a new `object` in JavaScript named `config`.
+
+**What is a Javascript Object?**
+
+Take the object closest to you right now, for most of you that might be a cell phone. A cell phone can have a lot of characteristics, or _properties_. One could be a screen.
 
-`properties` are values that we store by name in the `object` and then retrieve at a later time.
+How would you describe a screen to someone who hasn't seen it before? You can tell them how big that screen is. Javascript objects are bundles of information stored in this way: a property and a value. What would a Javascript object of our phone look like?
```

Comment:
> How would you describe a screen to someone who hasn't seen it before?

Does it matter if someone hasn't seen a screen in this example?

I can use js objects to describe things you've seen before too. For example:

```js
var me = {
  hair: 'black',
  eye: 'brown',
  height: '6 feet 4 inch'
}
```